src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets

```diff
@@ -92,7 +92,7 @@ Copyright (c) .NET Foundation. All rights reserved.
                              _CGenJit=$(_JitPath);
                              _CGenInputAssembly=%(_AssembliestoCrossGen.Fullpath);
                              _CGenOutputAssembly=$([System.IO.Path]::Combine($(_RuntimeOptimizedDir),%(RecursiveDir)%(Filename)%(Extension)));
-                             _CGenPlatformAssembliesPath=$(_NetCoreRefDir)$(PathSeparator)$(_RuntimeRefDir);" />
+                             _CGenPlatformAssembliesPath=$(_RuntimeRefDir)$(PathSeparator)$(_NetCoreRefDir);" />
```

Comment:
Can you explain to me why we are passing ALL the paths into the `/Platform_Assemblies_Paths` parameter?  Why are we not using `/App_Paths` ?  /cc @gkhanna79 

```
    /App_Paths <path[;path]>
                         - List of paths containing user-application assemblies and resources
    /Platform_Assemblies_Paths <path[;path]>
                         - List of paths containing target platform assemblies
                         - Cannot be used with Trusted_Platform_Assemblies
```